src/picongpu/include/simulation_defines/param/speciesDefinition.param

```diff
@@ -143,6 +173,9 @@ PIC_Ions
 typedef MakeSeq<
 Species1,
 Species2
+#if(ENABLE_SYNCHROTRON_PHOTONS == 1)
```

Comment:
Here, the precompiler statement is necessary because the `HasFlag` condition is based on a complete species list.